docs/userdoc.qmd

```diff
@@ -17,59 +24,113 @@ To browse through your data collections in iRODS click on the **Browser** tab.
    - Click on the `Name`  of a collection or option. This will load the tabs below the table (see below)
```

Comment:
The whole browser tab and its sub-tabs  have been significantly updated and this documentation should also be.  For instance, the _Resources_ tab is now the _Replicas_ tab and it shows much more information including the resource hierarchy and not just the name of the leaf resource.